### PR TITLE
feat: create-react-app and create-next-app templates

### PR DESCRIPTION
**Pull request by SiTaggart**
Create quick start guides.

Use create-react-app and create-next-app as bootstrap mechanisms for a Paste app.

Use the nextjs template as the code sandbox template too

### CODE REVIEW

File: packages/paste-cra-template/template/src/components/Loading.tsx [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import * as React from 'react';
+import {Spinner} from '@twilio-paste/core/spinner';
+import {Box} from '@twilio-paste/core/box';
+
+export const Loading: React.FC = () => {
+  return (
+    <Box as="div">
+      <Spinner decorative={false} title="Loadings" />
```
Comment (richbachman) [inline packages/paste-cra-template/template/src/components/Loading.tsx:8]:
```suggestion
      <Spinner decorative={false} title="Loading" />
```

File: packages/paste-nextjs-template/pages/index.tsx [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+      </Heading>
+
+      <UnorderedList>
+        <ListItem>
+          <Heading as="h3" variant="heading30">
+            <Anchor href="https://paste.twilio.design">
+              <Box as="span" display="flex" alignItems="center">
+                Paste Documentation <ArrowForwardIcon decorative size="sizeIcon60" />
+              </Box>
+            </Anchor>
```
Comment (richbachman) [inline packages/paste-nextjs-template/pages/index.tsx:40]: Could we just use the `showExternal` prop for these Anchors?

**Review by richbachman — APPROVED** [review]

File: .codesandbox/ci.json [hunk 3]
```diff
@@ -1,22 +1,67 @@
...
   "buildCommand": "build",
   "packages": [
-    "/packages/paste-core/components/*",
-    "/packages/paste-core/layout/*",
-    "/packages/paste-core/primitives/*",
-    "/packages/paste-core/core-bundle",
-    "/packages/paste-design-tokens",
     "/packages/paste-icons",
+    "/packages/paste-core/core-bundle",
+    "/packages/paste-core/components/alert",
```
Comment (TheSisb) [inline .codesandbox/ci.json:6]: Curious why you added the individual packages as well? Does it fix that "cant find X package" issue?

File: packages/paste-cra-template/template/src/reportWebVitals.ts [hunk 4]
```diff
@@ -0,0 +1,21 @@
+import {ReportHandler} from 'web-vitals';
```
Comment (TheSisb) [inline packages/paste-cra-template/template/src/reportWebVitals.ts:1]: `import type`

File: packages/paste-cra-template/template/README.md [hunk 5]
```diff
@@ -6,23 +8,23 @@ In the project directory, you can run:
 
 ### `yarn start`
 
-Runs the app in the development mode.<br />
+Runs the app in the development mode.\
```
Comment (TheSisb) [inline packages/paste-cra-template/template/README.md:11]: Accidental `\` in this file?

File: packages/paste-nextjs-template/package.json [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+    "build": "next build",
+    "start": "next start"
+  },
+  "dependencies": {
+    "@twilio-paste/core": "^4.2.4",
+    "@twilio-paste/icons": "^3.10.5",
+    "next": "^10.0.4",
+    "prop-types": "^15.7.2",
+    "react": "^16.8.6",
+    "react-dom": "^16.8.6"
```
Comment (TheSisb) [inline packages/paste-nextjs-template/package.json:16]: Should we add web-vitals to this template too?

File: packages/paste-website/src/pages/getting-started/engineering/index.mdx [hunk 7]
```diff
@@ -49,7 +49,33 @@ export const pageQuery = graphql`
 
 <content>
 
-## Installation
+## Quickstart
+
+The quickest way to get started with Paste is to bootstrap a new React app using one of our templates for popular React frameworks, [Next.js](https://nextjs.org/) and [Create React App](https://create-react-app.dev/).
+
+Both approaches will create a new React app with Paste installed and set up ready to start developing with.
```
Comment (TheSisb) [inline packages/paste-website/src/pages/getting-started/engineering/index.mdx:56]: `with Paste installed and ready for development.` maybe?

File: .codesandbox/ci.json (hunk 3 above)
Comment (SiTaggart) [reply .codesandbox/ci.json:6]: Yup, essentially. What was happening before was it was building core and icons, but using all the packages from npm. This way codesandbox hosts all packages on it's package manager, and when we add a new package, we'll be able to test it in there too on the PR

File: packages/paste-cra-template/template/README.md (hunk 5 above)
Comment (SiTaggart) [reply packages/paste-cra-template/template/README.md:11]: It's the markdown version of `<br />`

File: packages/paste-nextjs-template/package.json (hunk 6 above)
Comment (SiTaggart) [reply packages/paste-nextjs-template/package.json:16]: Might give it a try

Comment (SiTaggart) [reply packages/paste-nextjs-template/package.json:16]: Whoa, it's basically built in already https://nextjs.org/docs/advanced-features/measuring-performance

File: packages/paste-nextjs-template/pages/index.tsx (hunk 2 above)
Comment (SiTaggart) [reply packages/paste-nextjs-template/pages/index.tsx:40]: Fair

File: packages/paste-website/src/pages/getting-started/engineering/index.mdx (hunk 7 above)
Comment (SiTaggart) [reply packages/paste-website/src/pages/getting-started/engineering/index.mdx]: I dig it

**Review by TheSisb — APPROVED** [review]

**Pull request merged**